hack/install-tools.sh

```diff
@@ -36,7 +36,7 @@ _install_kustomize() {
 }
 
 _install_controller_gen() {
-	_install_tool sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2
+	_install_tool sigs.k8s.io/controller-tools/cmd/controller-gen@v0.14.0
```

Comment:
Looks fine. However, I tested with both `go1.20.4` and `go1.22.2` with clean build and I get also this (uncommitted) change:

```
$ git diff
diff --git a/api/v1alpha1/zz_generated.deepcopy.go b/api/v1alpha1/zz_generated.deepcopy.go
index 0336f1f..5713bf3 100644
--- a/api/v1alpha1/zz_generated.deepcopy.go
+++ b/api/v1alpha1/zz_generated.deepcopy.go
@@ -1,5 +1,4 @@
 //go:build !ignore_autogenerated
-// +build !ignore_autogenerated

```

(using centos9 stream).

Reply:
Done.